src/defaults.js

```diff
@@ -1,4 +1,4 @@
-import { TYPE_ELEMENT } from './constants';
+import binding from './polyfill/binding';
```

Comment:
From the naming of this file and its export, it's not obvious to me that it is only for elements — so either I'm not familiar with the code/w3c-spec enough (which is true) or it could use a more telling name.